daemon/logger/fluentd/fluentd_test.go

```diff
@@ -199,3 +205,118 @@ func TestValidateLogOptAddress(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateWriteTimeoutDuration(t *testing.T) {
+	invalidDurations := []string{"-1", "1", "-1s"}
+	for _, d := range invalidDurations {
+		t.Run("invalid "+d, func(t *testing.T) {
+			err := ValidateLogOpt(map[string]string{writeTimeoutKey: d})
+			assert.ErrorContains(t, err, "invalid value for fluentd-write-timeout:")
+		})
+	}
+
+	validDurations := map[string]time.Duration{
+		"100ms": 100 * time.Millisecond,
+		"10s":   10 * time.Second,
+		"":      0,
+	}
+	for k, v := range validDurations {
+		t.Run("valid "+k, func(t *testing.T) {
+			err := ValidateLogOpt(map[string]string{writeTimeoutKey: k})
+			assert.NilError(t, err)
+			// Ignore err here since that's already checked in the ValidateLogOpt error check.
+			cfg, _ := parseConfig(map[string]string{writeTimeoutKey: k})
```

Comment:
1-line comment to explain why we're not adding a 1-line assert; I guess we might as well just assert it's indeed not an error;

```go
cfg, err := parseConfig(map[string]string{writeTimeoutKey: k})
assert.NilError(t, err)
```